### ISSUE TITLE
Adjust behavior of Shift+Enter/Execute and select next cell.

### ISSUE THREAD

**Issue by lurgyman63** [ipython/ipython#8359]
Could Shift+Enter (execute and select next cell) execute and then skip to the next cell _of the same type_?

I often use this shortcut with the intent of executing a sequence of python cells, but it's less efficient to Shift+Enter through markdown cells when I'm trying to focus on code.

Likewise, if I'm editing a markdown cell, it's because I'm on a tear explaining the contents of the notebook, and am more likely to want to skip to the next markdown cell, not a code cell.

**Comment by Carreau** [comment]
It would be doable, you can monkey patch IPython with your own javascript logic in custom.js.

**Comment by lurgyman63** [comment]
I'll definitely look into that.  My 2 cents would be that it could be a sensible default, but I can definitely start playing with custom.js.

Thanks for the great tool!

**Comment by Carreau** [comment]
I doubt it would be a nice default, it would be confusing imho.

**Comment by lurgyman63** [comment]
OK, I guess that's that then :)

**Comment by Carreau** [comment]
If you figure out how to do it in custom.js feel free to share  though. I have a vague description on how to add / modify shortcut [here}(http://carreau.gitbooks.io/jupyter-book/content/keyboardshortcut.html) (PR welcomed)

**Comment by juhasch** [comment]
I have tried the idea of skipping markdown cells, and concur with @Carreau's opinion, I found it confusing. 
If you want to play with changing the key behavior, take a look here:
https://github.com/ipython-contrib/IPython-notebook-extensions/blob/3.x/usability/navigation-hotkeys/main.js

For example, for shift-enter, I only go into edit mode again, if the next cell is a codecell. This feels naturals and avoids unrendering markdown cells.
